variants/d1_mini32/pins_arduino.h

```diff
@@ -0,0 +1,33 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include <../d32/d32_core.h>
+
+static const uint8_t LED_BUILTIN = 2;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+static const uint8_t _VBAT = 35; // battery voltage
+
+#define PIN_WIRE_SDA SDA // backward compatibility
+#define PIN_WIRE_SCL SCL // backward compatibility
+
+static const uint8_t D0   = 26;
+static const uint8_t D1   = 22;
+static const uint8_t D2   = 21;
+static const uint8_t D3   = 17;
+static const uint8_t D4   = 16;
+static const uint8_t D5   = 18;
+static const uint8_t D6   = 19;
+static const uint8_t D7   = 23;
+static const uint8_t D8   = 5;
+static const uint8_t RXD  = 3;
+static const uint8_t TXD  = 1;
```

Comment:
I don't remember where I found pin numbers 9 and 10 for RXD0/TXD0 - RX and TX are in d32_core.h already. I am unsuccessful in redirecting Serial to vacate 3 and 1, in order to use EspSoftwareSerial for the USB logging port, and have all three HW UARTs freely available. Is this not supported, to redirect the pin assignment of Serial?

Reply:
if you make them to be defines, they will overwrite the default pins for the UARTs:
```cpp
//this will make Serial1.begin(baud) to run on pins 21 and 22
//add RX2/TX2 to overwrite Serial2
#define RX1 21
#define RX2 22
```

Reply:
Yes, thank you for explaining, I understand that from reading the code in HardwareSerial.(h|cpp) and esp32-hal-uart.(h|cpp).
But: the code suggests that Serial (0) can be redirected from 3 (RX) and 1 (TX), for the love of it, I can't that to work. Is the UART0 somehow hardwired to these ports and cannot be redirected, even if the code tries to, and fails silently?
As far as the PR is concerned, I've completed it, would you merge it now as it is?

Reply:
if you want to run Serial on other pins, you really need to specify them in Serial.begin
RX and TX are hardcoded to 1 and 3 otherwise as that is the real Serial port (like AVR arduinos and such)